### ISSUE TITLE
`use` to show the full long name

### ISSUE THREAD

**Issue by mvuets** [gugod/App-perlbrew#439]
I often switch among few perl@lib installations. And I find it a bit annoying that argument-less `perlbrew use` does not print a full long name (one with `@lib` suffix), so I need to resort to `perlbrew list` and look for like marked with `*`:
```
$ perlbrew use
Currently using perl-5.20.1
# ^ that is not very accurate, because this...
$ perlbrew list | grep '^\*'
* perl-5.20.1@main
```

Does it make sense to make `perlbrew use` print out a full name of an active installation?

**Comment by mstreuhofer** [comment]
fixed in #442

**Comment by mvuets** [comment]
Sweet. Thanks @mstreuhofer!
